aiohttp/web_reqrep.py

```diff
@@ -72,6 +77,11 @@ def content_length(self, _CONTENT_LENGTH=hdrs.CONTENT_LENGTH):
 FileField = collections.namedtuple('Field', 'name filename file content_type')
 
 
+class ContentCoding(enum.Enum):
+    deflate = 1
```

Comment:
Also would like to note that this list of encodings [is not complete](http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.5). Never saw `compress` in the wild, however, but `identity` is pretty common one.